### ISSUE TITLE
Inconsistent accessibility

### ISSUE THREAD

**Issue by VargaDot** [SP4R0W/Composer-Godot#32]
Just by building the project, 2 errors pop up
```
CS7025: Inconsistent accessibility: event type 'Scene.FinishedLoadingEventHandler' is less accessible than event 'Scene.FinishedLoading' D:\Godot Projects\Composer-Godot\Godot.SourceGenerators\Godot.SourceGenerators.ScriptSignalsGenerator\ComposerLib.Scene_ScriptSignals.generated.cs(38,72)
```
```
CS7025: Inconsistent accessibility: event type 'Scene.FinishedCreatingEventHandler' is less accessible than event 'Scene.FinishedCreating' D:\Godot Projects\Composer-Godot\Godot.SourceGenerators\Godot.SourceGenerators.ScriptSignalsGenerator\ComposerLib.Scene_ScriptSignals.generated.cs(44,73)
```

**Comment by VargaDot** [comment]
note: changing the accessibility parameter of both signals from `internal` to **anything else** does NOT fix it (brute force failed no way)
